imagescan/delta.go

```diff
@@ -293,6 +293,35 @@ func (d *deltaState) findBestNode(nodeNames []string, requiredMemory *inf.Dec, r
 	return candidates[0].name, nil
 }
 
+func (d *deltaState) nodeNum() int {
+	d.mu.RLock()
+	defer d.mu.RUnlock()
+
+	return len(d.nodes)
+}
+
+func (d *deltaState) randomNodeName() string {
```

Comment:
why is this called random? 😆

Reply:
this is fetching key from hash map, there are no guarantees